### PR TITLE
Changes to discovery doc sample generator, mostly stylistic.

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

**Comment by broady** [comment]
@jba @pongad @okdave PTAL

I couldn't figure out how to run the generator or see the output, so not sure what it'll look like.

The paging code seems a bit gross. Let me know what it could look like.

Baseline here: https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/go_library.baseline

**Comment by broady** [comment]
DO NOT SUBMIT (need to fix the tests)

File: src/test/java/com/google/api/codegen/testdata/discoveries/go/go_appengine.v1beta5.json.baseline [hunk 1]
```diff
@@ -315,47 +288,43 @@ import (
...
 
-    requestBody = &appengine.Service{}
-  )
+  // Part of `name`. See documentation of `appsId`.
+  servicesId := "" // TODO: Update placeholder value.
 
-  response, err := client.Apps.Services.Patch(appsId, servicesId, requestBody).Context(ctx).Do()
+  resp, err := client.Apps.Services.Patch(appsId, servicesId, &appengine.Service{
+                                            // TODO: Fill required fields.
+                                          }).Context(ctx).Do()
```
Comment (pongad) [inline src/test/java/com/google/api/codegen/testdata/discoveries/go/go_appengine.v1beta5.json.baseline]: The indentation is rather unfortunate, but we can gofmt this out later.

**Comment by pongad** [comment]
lgtm

**Comment by broady** [comment]
Please merge! Rebasing is getting tedious :)

**Pull request merged**